src/System.Drawing.Primitives/System.Drawing.Primitives.targets

```diff
@@ -0,0 +1,27 @@
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="_BuildSystemDrawingPrimitivesFacade">
+   <MSBuild Projects="..\Mono.Android\Mono.Android.csproj"
+             Targets="GetTargetPath"
+             Properties="Configuration=$(Configuration)">
+        <Output TaskParameter="TargetOutputs"
+                ItemName="MonoAndroidAssembly" />
+    </MSBuild>
+    <Exec
+        Command="make -C $(MonoSourceFullPath)\mcs\class\Facades\System.Drawing.Primitives PROFILE=monodroid EXTRA_LIB_MCS_FLAGS=&quot;-r:%(MonoAndroidAssembly.Identity)&quot;"
+        WorkingDirectory="$(MonoSourceFullPath)"
```

Comment:
I don't think that `WorkingDirectory` is required, as `make -C` will *also* change the working directory as well. Thus, this seems redundant.